### PR TITLE
Add dgrisonnet to custom-metrics-apiserver and prometheus-adapter maintainers

### PR DESCRIPTION
**Pull request by dgrisonnet**
/cc @s-urbaniak

### CODE REVIEW

**Comment by s-urbaniak** [comment]
leaving final pair of eyes to @ehashman :eyes:

File: config/kubernetes-sigs/sig-instrumentation/teams.yaml
Comment (ehashman) [inline config/kubernetes-sigs/sig-instrumentation/teams.yaml:53]: Can we get this to match the OWNERS file?

Should be

```
- dgrisonnet
- lilic
- s-urbaniak
```

**Review by ehashman — CHANGES_REQUESTED** [review]

File: config/kubernetes-sigs/sig-instrumentation/teams.yaml
Comment (dgrisonnet) [reply config/kubernetes-sigs/sig-instrumentation/teams.yaml:53]: Should we also add `s-urbaniak` to the maintainers, even though he is already in the admins list?

Comment (ehashman) [reply config/kubernetes-sigs/sig-instrumentation/teams.yaml:53]: Yes, he should be in both GitHub teams.

**Comment by dgrisonnet** [comment]
Seems like the tests are failing because @lilic isn't part of the kubernetes-sigs org

**Comment by lilic** [comment]
Feel free to remove me from the OWNERS file in that case, I do not plan on contributing going forward. Thank you!

**Comment by dgrisonnet** [comment]
Thank you for the heads-up @lilic

I updated the PR, @ehashman can you please take another look?

File: config/kubernetes-sigs/sig-instrumentation/teams.yaml
Comment (ehashman) [reply config/kubernetes-sigs/sig-instrumentation/teams.yaml:53]: Still missing Lili.

**Review by ehashman — COMMENTED** [review]
/hold cancel
/approve
/lgtm

**Pull request merged**
